core/src/main/java/org/jboss/jca/core/connectionmanager/pool/mcp/SemaphoreArrayListManagedConnectionPool.java

```diff
@@ -698,9 +701,9 @@ public void removeIdleConnections()
       {
          for (ConnectionListener cl : destroy)
          {
-            log.tracef("Destroying timedout connection %s", cl);
```

Comment:
I'm not sure why the log message is different here... maybe the other branch had a different message, or did I add this difference in the log messages without realizing it, in the original commit?